### PR TITLE
Updating folders

### PR DESCRIPTION
**Pull request by angela4713**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Angela Guette, Wiebke Frey, Laura Garcia
```
Comment (padde) [inline LICENSE.txt]: Should be Homeday, not you personally

Comment (padde) [inline LICENSE.txt]: Should be Homeday, not you personally

File: immobilienscout.gemspec [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+require 'immobilienscout/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'immobilienscout'
+  spec.version       = Immobilienscout::VERSION
+  spec.authors       = ['Angela Guette', 'Wiebke Frey', 'Laura Garcia']
+  spec.email         = ['backend-team@homeday.de']
+
+  spec.summary       = 'A ruby gem to handle Immobilienscout API'
+  spec.description   = 'This gem provides HTTP endpoints to connect your app with Immobilienscout API'
```
Comment (padde) [inline immobilienscout.gemspec]: It does not provide HTTP _endpoints_, it provides a HTTP _client_.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,6 @@
 # Immobilienscout Changelog
 
-## v0.0.3
-
-Update the response accord to the Immobilienscout changes in the `JSON` structure
-
-## v0.0.3
+## v0.0.2
```
Comment (padde) [inline CHANGELOG.md]: Are there a few points missing here? For now it should not be commited the new version number, instead the first header should be something like "unreleased" until we really release and tag the new version. This should be a separate commit on master.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.5.1
```
Comment (padde) [inline .travis.yml]: If we're testing for 2.5.x we should test the latest minor release 2.5.5. Even better, we can leverage the Travis build matrix to test our gem against all the [currently maintained ruby versions](https://www.ruby-lang.org/en/downloads/branches/)

```
rvm:
  - 2.4.6
  - 2.5.5
  - 2.6.3
```

File: immobilienscout.gemspec [hunk 5]
```diff
@@ -1,17 +1,36 @@
...
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'json'
+  spec.add_dependency 'multipart-post'
+  spec.add_dependency 'activesupport'
+  spec.add_dependency 'webmock'
```
Comment (padde) [inline immobilienscout.gemspec]: Why is webmock a production dependency? should be dev only.

File: immobilienscout.gemspec [hunk 6]
```diff
@@ -1,17 +1,36 @@
...
+  spec.add_development_dependency 'bundler', '~> 2.0'
+  spec.add_development_dependency 'rake', '~> 10.0'
```
Comment (padde) [inline immobilienscout.gemspec]: Gemfile.lock has Rake 12.x, why are we downgrading now?

File: lib/immobilienscout/version.rb [hunk 7]
```diff
@@ -0,0 +1,3 @@
+module Immobilienscout
+  VERSION = '0.0.2'
+end
```
Comment (padde) [inline lib/immobilienscout/version.rb]: Version bump should usually be a separate commit on `master`, which can then be tagged for release.

For example: https://github.com/plataformatec/devise/commit/2e5b5fcd705b06c518ab0156b96badb91c4cb6ea

File: LICENSE.txt (hunk 1 above)
Comment (padde) [reply LICENSE.txt]: Maybe also for the gemspec authors? Better ask mgmt what's right here 😉

File: immobilienscout.gemspec (hunk 2 above)
Comment (angela4713) [reply immobilienscout.gemspec]: Right!

File: immobilienscout.gemspec (hunk 6 above)
Comment (angela4713) [reply immobilienscout.gemspec]: This was generated by `Bundler` 🤔 I will fix it here

File: lib/immobilienscout/version.rb (hunk 7 above)
Comment (angela4713) [reply lib/immobilienscout/version.rb]: Yes but this is not a bump, is the actual version

File: LICENSE.txt (hunk 1 above)
Comment (angela4713) [reply LICENSE.txt]: Homeday sounds fine ☺️

File: .travis.yml [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
```
Comment (padde) [inline .travis.yml]: 2.5.1. is not necessary if we test 2.5.5...

File: LICENSE.txt [hunk 9]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Homeday
```
Comment (padde) [inline LICENSE.txt]: Should be full legal name of company, `Homeday GmbH`

File: immobilienscout.gemspec [hunk 10]
```diff
@@ -1,17 +1,36 @@
...
-  s.add_dependency 'activesupport'
-  s.add_dependency 'webmock'
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'immobilienscout/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'immobilienscout'
+  spec.version       = Immobilienscout::VERSION
+  spec.authors       = ['Homeday']
```
Comment (padde) [inline immobilienscout.gemspec]: Also `Homeday GmbH`

File: lib/immobilienscout/version.rb (hunk 7 above)
Comment (padde) [reply lib/immobilienscout/version.rb]: Ah right makes sense!

File: CHANGELOG.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
+# Immobilienscout Changelog
+
+## v0.0.3 [Unreleased]
+
```
Comment (padde) [inline CHANGELOG.md]: So this upcoming 0.3.3 release already includes a few changes that were not released yet right? The should be in here already. In the future, the changelog should be updated directly in the PR where changes are made.

File: immobilienscout.gemspec [hunk 12]
```diff
@@ -1,17 +1,36 @@
...
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'json'
+  spec.add_dependency 'multipart-post'
+  spec.add_dependency 'activesupport'
+  spec.add_development_dependency 'webmock', '~> 3.6'
+  spec.add_development_dependency 'bundler', '~> 2.0'
+  spec.add_development_dependency 'rake', '~> 12.0'
+  spec.add_development_dependency 'rspec', '~> 3.0'
```
Comment (padde) [inline immobilienscout.gemspec]: We still duplicate lots of this in `Gemfile`. As commented before, the `Gemfile` should not include any actual dependencies, just the line `gemspec`

File: spec/spec_helper.rb [hunk 13]
```diff
@@ -60,8 +60,12 @@
   # inherited by the metadata hash of host groups and examples, rather than
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
-  # VCR cassetes config
 
+  config.after :all do
+    Timecop.return
+  end
```
Comment (padde) [inline spec/spec_helper.rb]: This should not be necessary. It is a big source of problems in some of our apps. Please use [safe mode](https://github.com/travisjeffery/timecop#timecopsafe_mode) instead.

File: .travis.yml (hunk 8 above)
Comment (angela4713) [reply .travis.yml]: oh I see

File: CHANGELOG.md (hunk 11 above)
Comment (angela4713) [reply CHANGELOG.md]: Yes, this version will have the new JSON structure response for the ScoutReport from Immobilienscout. I will add it here then.

File: immobilienscout.gemspec (hunk 12 above)
Comment (angela4713) [reply immobilienscout.gemspec]: Good! but you didn't leave any comment about this before 🤔 😅

Comment (padde) [reply immobilienscout.gemspec]: Maybe you overlooked it 🙂 

Screenshot 2019-08-15 at 14 41 13

Comment (angela4713) [reply immobilienscout.gemspec]: Ahh ok haha it's because I normally look the individual ones not the main one 😂

Comment (padde) [reply immobilienscout.gemspec]: Yeah usually the main comment is just small talk, true 😆
